cli/internal/helm/loader_test.go

```diff
@@ -59,25 +60,34 @@ func TestConstellationServices(t *testing.T) {
 		cnmImage           string
 	}{
 		"GCP": {
-			config: &config.Config{Provider: config.ProviderConfig{GCP: &config.GCPConfig{
-				DeployCSIDriver: func() *bool { b := true; return &b }(),
-			}}},
+			config: &config.Config{
+				AttestationVariant: oid.GCPSEVES{}.String(),
+				Provider: config.ProviderConfig{GCP: &config.GCPConfig{
+					DeployCSIDriver: func() *bool { b := true; return &b }(),
```

Comment:
Could define and use [toPtr[T any](v T) *T](https://github.com/edgelesssys/constellation/blob/8679988b6c6989affd330ab07dc924c164b0202e/internal/kubernetes/configmaps.go#L60-L62).

Reply:
This is not code I wrote for this PR, but yes, it can be changed